clients/admin-ui/src/features/config-wizard/PrivacyDeclarationForm.tsx

```diff
@@ -0,0 +1,387 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  FormLabel,
+  Heading,
+  Stack,
+  Text,
+  Tooltip,
+  useToast,
+} from "@fidesui/react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/dist/query/fetchBaseQuery";
+import { Form, Formik } from "formik";
+import type { NextPage } from "next";
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { AddIcon, QuestionIcon } from "~/features/common/Icon";
+import {
+  selectDataQualifier,
+  setDataQualifier,
+  useGetDataQualifierQuery,
+} from "~/features/data-qualifier/data-qualifier.slice";
+import {
+  selectDataSubjects,
+  setDataSubjects,
+  useGetAllDataSubjectsQuery,
+} from "~/features/data-subjects/data-subject.slice";
+import {
+  selectDataUse,
+  setDataUse,
+  useGetDataUseQuery,
+} from "~/features/data-use/data-use.slice";
+import {
+  selectDataCategories,
+  setDataCategories,
+  useGetAllDataCategoriesQuery,
+} from "~/features/taxonomy/data-categories.slice";
+
+import {
+  CustomMultiSelect,
+  CustomSelect,
+  CustomTextInput,
+} from "../common/form/inputs";
+import { isErrorWithDetail, isErrorWithDetailArray } from "../common/helpers";
+import {
+  useGetSystemByFidesKeyQuery,
+  useUpdateSystemMutation,
+} from "../system/system.slice";
+
+interface PrivacyDeclaration {
```

Comment:
why not use this? https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/system/types.ts#L29-L36